### DISCUSSION TITLE
Termine duplizieren können / Termine als Beiträge veröffentlichen

### DISCUSSION THREAD

**Discussion by janschreier** [codeispoetry/sunflower#775] [Ideen und Wünsche]
Aus unserem Bürgermeisterwahlkampf sind mir zwei Wünsche übriggeblieben für das aktuelle Theme:
1. Wir haben ein Terminformat, dass wir an vielen Tagen angeboten haben. D. h. ich durfte den Text jedes Mal kopieren und auch Ort und Uhrzeit immer von Hand eintragen. Hier wäre ein "kopieren" Button extrem praktisch. Da wäre es mir schon genug, wenn im Titel "Kopie" am Anfang stehen würde und der Termin als "nicht veröffentlicht" gekennzeichnet ist. Also sonst zur gleichen Uhrzeit/Tag eingetragen ist. 

2. Wenn ich Termine anlegen, schreibe ich dazu fast immer 1:1 eine News, um den Termin auch zu bewerben. Hier wäre es schön, wenn man Termine auch direkt in den News/Beiträgen mitanzeigen könnte. 

Ich hoffe der Wunsch ist so verständlich, falls nicht, sagt gern bescheid. Dann bebildere ich meine Wünsche noch etwas mehr :)

**Comment by FloJuh** [comment]
Punkt 1 geht aktuell über die Einbindung eines externen Kalenders

**Comment by ediordna** [comment]
Zu Punkt 1: Ginge das Erstellen eines Termins auch über die REST-API? 
`curl -X POST "https://gruene-xyz.de/wp-json/wp/v2/sunflower_event" \
  -H "Authorization: Basic BASE64(admin:ApplicationPassword)" \
  -H "Content-Type: application/json" \
  -d '{
    "title": "Mein Termin",
    "content": "Beschreibung des Termins",
    "status": "publish",
    "meta": {
      "start_date": "2026-03-01 18:00:00",
      "end_date": "2026-03-01 20:00:00",
      "location": "Veranstaltungsort",
      "street": "Straße 12",
      "city": "Musterhausen"
    },
    "tags": [12, 34]
  }'`
Wie würden die Metafelder im sunflower-Theme denn heißen? Ich hab im Code und Backend versucht es rauszukriegen, war aber nicht erfolgreich.
